node/cn/backend.go

```diff
@@ -233,6 +234,7 @@ func New(ctx *node.ServiceContext, config *Config) (*CN, error) {
 		MaxBlockGasUsedForBaseFee: governance.MaxBlockGasUsedForBaseFee(),
 		BaseFeeDenominator:        governance.BaseFeeDenominator(),
 	}
+	//
```

Comment:
```suggestion
```
delete unused comment

Reply:
@hqjang-pepper I intended this comment to discuss continue with @aidan-kwon.
Please look at https://github.com/klaytn/klaytn/pull/1379#discussion_r885456549
I will remove it when discussion is done